config/routes.rb

```diff
@@ -4,4 +4,9 @@
     logger.level = Logger.const_get(::Settings.log.level_websocket.upcase)
     mount WebsocketServer.new(:logger => logger) => '/ws'
   end
+
+  mount ManageIQ::GraphQL::Engine, :at => '/graphql'
```

Comment:
These routes should be in the engine itself, not here.  That way if we don't include the gem, then we don't load the routes.

Reply:
Ugh, I really liked the explicitness here, letting this application decide how it routes things. Still, I see your point about not including the routes given the Bundler group work (@jrafanie) so I can see this is going to be argument I will surely lose :) I'll change it.